extide/gradle/test/unit/src/org/netbeans/modules/gradle/api/execute/GradleCommandLineTest.java

```diff
@@ -304,4 +305,22 @@ public void testJVMArgs2() throws IOException {
         GradleCommandLine.addGradleSettingJvmargs(root.getRoot(), jvmargs);
         assertEquals(Arrays.asList("-Dfile.encoding=UTF-8", "-Dsomething=space value"), jvmargs);
     }
+    
+    @Test
+    public void testDescriptions() {
+        Set<GradleCommandLine.GradleOptionItem> all = new HashSet<>();
+        all.addAll(Arrays.asList(GradleCommandLine.Flag.values()));
+        all.addAll(Arrays.asList(GradleCommandLine.Parameter.values()));
+        all.addAll(Arrays.asList(GradleCommandLine.Property.values()));
+        all.remove(GradleCommandLine.Parameter.IMPORT_BUILD);
+        List<GradleCommandLine.GradleOptionItem> missing = new ArrayList<>();
+        for (GradleCommandLine.GradleOptionItem item : all) {
+            try {
+                assertNotNull(item.getDescription());
+            } catch (MissingResourceException ex){
+                missing.add(item);
+            }
+        }
+        assertTrue(missing.toString(), missing.isEmpty());
```

Comment:
Perfect ! Ensures that the desc is filled if an option / parameter is added in the future.